packages/storybook/stories/components/tabs/technical-information.mdx

```diff
@@ -0,0 +1,40 @@
+import { Canvas, Meta, Markdown } from '@storybook/blocks';
+import SpecificationsTabs from '@ovhcloud/ods-components/src/components/tabs/documentation/spec.md?raw';
+import { Banner } from '../../banner';
+import * as TabsStories from './tabs.stories';
+
+<Meta of={ TabsStories } name='Technical information' />
+
+<Banner of={ TabsStories } />
+
+## Overview
+
+<Canvas of={ TabsStories.Overview } sourceState='none' />
+
+<Markdown>{ SpecificationsTabs }</Markdown>
+
+## Style customization
+
+You can apply your own style on the whole tabs container or on each tab using css classes.
```

Comment:
You can apply your own style on the whole tabs container or on each tab using CSS classes.